code/game/objects/effects/effect_system/effects_smoke.dm

```diff
@@ -250,7 +250,7 @@
 	var/fraction = 1/initial(lifetime)
 	reagents.copy_to(C, fraction*reagents.total_volume)
 	reagents.reaction(M, INGEST, fraction)
-	log_attack("Smoke was breathed in by [M] with reagents [english_list(reagents.reagent_list)] at [AREACOORD(src)]")
+	M.log_message("Smoke was breathed in with reagents [english_list(reagents.reagent_list)]", LOG_ATTACK, null, FALSE) // Do not log globally b/c spam
```

Comment:
```suggestion
	M.log_message("breathed in some smoke with reagents [english_list(reagents.reagent_list)]", LOG_ATTACK, null, FALSE) // Do not log globally b/c spam
```

Mob logging text generally is first person past indicative (or passive) sentence minus the subject.